controller/src/beerocks/master/son_actions.cpp

```diff
@@ -312,6 +312,10 @@ void son_actions::handle_dead_node(std::string mac, std::string hostap_mac, db &
     if (parent_hostap_mac == hostap_mac) {
         if (mac_type == beerocks::TYPE_IRE_BACKHAUL || mac_type == beerocks::TYPE_CLIENT) {
             database.set_node_state(mac, beerocks::STATE_DISCONNECTED);
+
+            // Clear node ipv4
+            database.set_node_ipv4(mac);
```

Comment:
nitpick: wouldn't a call to an API  "clear_node_ipv4" be more readable to what is performed here?

Reply:
maybe, I just didn't want to add an additional function.